notification/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:12.18.2-alpine
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 8080
+CMD [ "node", "src/server.js" ]
```

Comment:
better to use some runner which can handle the error in better way like nodemon, pm2